Python/sysmodule.c

```diff
@@ -2264,12 +2264,14 @@ sys_activate_stack_trampoline_impl(PyObject *module, const char *backend)
 /*[clinic end generated code: output=5783cdeb51874b43 input=a12df928758a82b4]*/
 {
 #ifdef PY_HAVE_PERF_TRAMPOLINE
-#ifdef _Py_JIT
-    _PyOptimizerObject* optimizer = _Py_GetOptimizer();
-    if (optimizer != NULL) {
-        Py_DECREF(optimizer);
-        PyErr_SetString(PyExc_ValueError, "Cannot activate the perf trampoline if the JIT is active");
-        return NULL;
+#if defined(_Py_TIER2) && (_Py_TIER2 % 2 != 0)
+    if (_PySys_JITEnabled()) {
+        _PyOptimizerObject* optimizer = _Py_GetOptimizer();
```

Comment:
```suggestion
        _PyOptimizerObject *optimizer = _Py_GetOptimizer();
```

The code style in this file seems to prefer this.